app/public/consoleWindow.js

```diff
@@ -0,0 +1,92 @@
+const consoleWindowElement = document.getElementById("consoleWindow");
+
+const lineHolderElement = document.createElement("div");
+lineHolderElement.className = "lineHolder";
+consoleWindowElement.appendChild(lineHolderElement);
+
+const scrollElement = consoleWindowElement;
+
+const snapToBottomTolerance = 50; //When within this many pixels from the bottom of the console, new messages will cause the scroll to snap to the bottom
+let modes = ["public", "private", "important", "critical"];
+let seconds = 60*4; //for testing
+
+//Expects an object with 3 keys: time, message, classString
+function AddConsoleLine(lineObj){
```

Comment:
I'm gonna come up with a way to standardize the data being passed between frontend and backend. I have an example in the playersetup PR here https://github.com/RichardLam772001/cs375/pull/29 with the use of a PlayerRoomUpdateData object. So these comments won't be necessary soon as long as we create the data classes